app/admin/user.rb

```diff
@@ -66,12 +69,8 @@
     active_admin_comments
   end
 
-  action_item :show, if: proc { !member.confirmed? } do
-    link_to 'Confirm member', confirm_admin_member_path(member), method: :put
-  end
-
-  action_item :show, if: proc { member.confirmed? } do
-    link_to 'Unconfirm member', unconfirm_admin_member_path(member), method: :put
+  action_item :show, if: proc { !user.confirmed? } do
+    link_to 'Confirm user', confirm_user_url(id: user.id, token: user.confirmation_token), method: :get
```

Comment:
Don't send them to this url. They should be redirected to the admin confirm action.